frontend/src/pages/Projects/Project/ChangeRequest/FormConfig.ts

```diff
@@ -99,12 +101,17 @@ export const FormConfig = (query: UseQueryResult<AxiosResponse, unknown>) => {
       required: true,
     },
     {
-      //TODO: Broken, wil be updated in future ticket
       fieldName: "types",
-      fieldType: "multiText",
+      fieldType: "autocompleteTable",
+      pickerName: "cr_type_option",
       fieldLabel: "Types",
       width: "full",
+      multiple: true,
       // required: true,
```

Comment:
I know this commented line was here before, but can it be removed or made false instead?

Reply:
yes technically, the next PR I'm fixing it